.evergreen/config.yml

```diff
@@ -98,7 +98,7 @@ functions:
             # If this was a patch build, doing a fresh clone would not actually test the patch
             cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
           else
-            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+            git clone --branch add-reviewers https://github.com/blink1073/drivers-evergreen-tools.git $DRIVERS_TOOLS
```

Comment:
could we have an `add-reviewers` branch directly tied to drivers-evergreen-tools?

Reply:
I'm going to add docs and open a PR once https://github.com/mongodb-labs/drivers-evergreen-tools/pull/411 is merged.